api/client/service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message SendCommandRequest {
+  enum CommandType {
+    kStart = 0;
+    kUpdate = 1;
+    UNSET = 2;
+  }
+  CommandType command_type = 1;
+  CommandLineOptions options = 2;
+}
+
+message SendCommandResponse {
+  repeated Output output = 1;
+}
+
+service NighthawkService {
+
+  /*
+  Accepts a SendCommandRequest, which can be used to initiate, update, or cancel a test.
```

Comment:
`SendCommand` sounds like a verb to me, whereas the request/response messages are nouns. I think maybe just calling them `ExecutionRequest` and `ExecutionResult` would be fine.

Reply:
Done